### PR TITLE
Release 7.1

### PR DESCRIPTION
**Pull request by SeanFeldman**
Connect to #79 

Update major version of Azure Storage library

### CODE REVIEW

**Comment by SeanFeldman** [comment]
Blocked until all other packages (ASQ, ASP) are upgraded and released.

File: src/NServiceBus.Hosting.Azure.HostProcess/packages.config [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
-  <package id="NuGetPackager" version="0.6.0" targetFramework="net452" developmentDependency="true" />
-  <package id="System.Spatial" version="5.7.0" targetFramework="net452" />
-  <package id="WindowsAzure.Storage" version="7.2.1" targetFramework="net452" />
+  <package id="GitVersionTask" version="3.6.5" targetFramework="net452" developmentDependency="true" />
+  <package id="Microsoft.Azure.KeyVault.Core" version="2.0.4" targetFramework="net452" />
+  <package id="Microsoft.Data.Edm" version="5.8.2" targetFramework="net452" />
+  <package id="Microsoft.Data.OData" version="5.8.2" targetFramework="net452" />
+  <package id="Microsoft.Data.Services.Client" version="5.8.2" targetFramework="net452" />
+  <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net452" />
+  <package id="NServiceBus" version="6.1.3" targetFramework="net452" />
```
Comment (boblangley) [inline src/NServiceBus.Hosting.Azure.HostProcess/packages.config]: Is there any way we can leave this at 6.0.0 to follow the nuspec [6.0.0, 7.0.0) dependency?

File: src/NServiceBus.Hosting.Azure/packages.config [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
-  <package id="Newtonsoft.Json" version="6.0.8" targetFramework="net461" />
-  <package id="NServiceBus" version="6.0.0" targetFramework="net452" />
-  <package id="NuGetPackager" version="0.6.0" targetFramework="net452" developmentDependency="true" />
+  <package id="GitVersionTask" version="3.6.5" targetFramework="net452" developmentDependency="true" />
+  <package id="Microsoft.Azure.KeyVault.Core" version="2.0.4" targetFramework="net452" />
+  <package id="Microsoft.Data.Edm" version="5.8.2" targetFramework="net452" />
+  <package id="Microsoft.Data.OData" version="5.8.2" targetFramework="net452" />
+  <package id="Microsoft.Data.Services.Client" version="5.8.2" targetFramework="net452" />
+  <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net452" />
+  <package id="NServiceBus" version="6.1.3" targetFramework="net452" />
```
Comment (boblangley) [inline src/NServiceBus.Hosting.Azure/packages.config]: Similar to the other, can we drop this back down to 6.0.0?

**Comment by SeanFeldman** [comment]
@boblangley done

**Comment by Scooletz** [comment]
This requires an additional review in regards to the Storage SDK version which is set to:
` <dependency id="WindowsAzure.Storage" version="[8.0.0, 9.0.0)" />`
and possibly could be 
` <dependency id="WindowsAzure.Storage" version="[7.0.0, 9.0.0)" />`

**Comment by SeanFeldman** [comment]
At the sync we agreed to release a hotfix (`7.1.1`) for the range issue. It will be a separate PR.

**Pull request merged**